examples/basic_crud.py

```
@@ -43,7 +44,9 @@ def main() -> None:
             article_id = response.get("data", {}).get("id")
             print(f"Created article with ID: {article_id}")
         except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)